docs/requirement-sets/requirement-sets.md

```diff
@@ -20,8 +20,8 @@ To work as expected, your Office Add-in might depend on a specific Office applic
 |Office 2016 on Windows<br>(volume-licensed perpetual)|<ul><li>TaskPane</li><li>Content</li></ul>|<ul><li>[ExcelApi 1.1](excel/excel-api-1-1-requirement-set.md)</li></ul>|<ul><li>[BindingEvents](common/office-add-in-requirement-sets.md#bindingevents)</li><li>[CompressedFile](common/office-add-in-requirement-sets.md#compressedfile)</li><li>[DialogApi 1.1](common/dialog-api-requirement-sets.md)\*</li><li>[DocumentEvents](common/office-add-in-requirement-sets.md#documentevents)</li><li>[File](common/office-add-in-requirement-sets.md#file)</li><li>[ImageCoercion 1.1](common/image-coercion-requirement-sets.md#imagecoercion-11)</li><li>[MatrixBindings](common/office-add-in-requirement-sets.md#matrixbindings)</li><li>[MatrixCoercion](common/office-add-in-requirement-sets.md#matrixcoercion)</li><li>[Selection](common/office-add-in-requirement-sets.md#selection)</li><li>[Settings](common/office-add-in-requirement-sets.md#settings)</li><li>[TableBindings](common/office-add-in-requirement-sets.md#tablebindings)</li><li>[TableCoercion](common/office-add-in-requirement-sets.md#tablecoercion)</li><li>[TextBindings](common/office-add-in-requirement-sets.md#textbindings)</li><li>[TextCoercion](common/office-add-in-requirement-sets.md#textcoercion)</li></ul>|
```

Comment:
I'm unable to add a comment to the Office on Mac row of the Excel table. We should also add RibbonApi 1.2 there.